packages/python-packages/api-stub-generator/apistub/nodes/_class_node.py

```diff
@@ -156,16 +178,20 @@ def _inspect(self):
                 members = inspect.getmembers(self.obj)
         else:
             members = inspect.getmembers(self.obj)
+
+        overloads = self._parse_overloads()
```

Comment:
I don't think this is an issue right now - but potentially if we had overloads on a standalone function (i.e. not a classmethod) they wouldn't be detected here?

Reply:
You are 100% correct.

Reply:
If we wanted to go the route of looking for standalone overloads I could, but it would be messier and I figured if it's not a credible scenario I wouldn't prioritize it.

Reply:
Yeah I don't think I'd prioritize it. However it might be worth sticking in a comment here with a note on how this would have to change if we found ourselves needing to support it.

Reply:
Added the note to the method docstring.